sui-execution/latest/sui-move-natives/src/dynamic_field.rs

```diff
@@ -498,7 +498,7 @@ pub fn has_child_object_with_ty(
     ))
 }
 
-fn get_tag_and_layouts(
+pub(crate) fn get_tag_and_layouts(
```

Comment:
maybe we should move this somewhere? 
And probably also want to doc comment now that it is public (which I should have really done even with it being private but... now you can do it! :P)

Reply:
Great idea! I'll move it up into `lib.rs`.